core/src/components/popover/test/size/popover.e2e.ts

```diff
@@ -8,26 +8,38 @@ import { openPopover, screenshotPopover } from '../test.utils';
  */
 configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
   test.describe(title('popover: size'), async () => {
-    /**
-     * The popovers have showBackdrop=false so we can open most of them at once
-     * and massively cut down on screenshots taken. The content has its own
-     * backdrop so you can still see the popovers.
-     */
-    // TODO FW-3598
-    test.skip('should calculate popover width based on sizing method', async ({ page }) => {
+    test.beforeEach(async ({ page }) => {
       await page.goto('/src/components/popover/test/size', config);
-      await page.setViewportSize(Viewports.tablet.portrait); // avoid popovers overlapping
+    });
+
+    test.describe('popover: invisible backdrop', async () => {
+      /**
+       * The popovers have showBackdrop=false so we can open most of them at once
+       * and massively cut down on screenshots taken. The content has its own
+       * backdrop so you can still see the popovers.
+       */
+      test('should calculate popover width based on sizing method', async ({ page }) => {
+        await page.setViewportSize(Viewports.tablet.portrait); // avoid popovers overlapping
```

Comment:
Is this code working? It looks like the screenshot tests aren't changing the viewport anymore: https://github.com/ionic-team/ionic-framework/pull/27686/files?diff=split&w=0#diff-3c55b3a487ce73e53d5c4295708ec8aafeb57b8e3709ab2124f603ca1ee14be1

Reply:
It is, this test is for the the existing screenshots that display the other popovers:

https://github.com/ionic-team/ionic-framework/blob/23165cdb8cf660f701d765194e5f035b82c6d2f1/core/src/components/popover/test/size/popover.e2e.ts-snapshots/popover-size-ios-ltr-Mobile-Chrome-linux.png

Since there aren't any differences, there weren't any changes.

This [test](https://github.com/ionic-team/ionic-framework/pull/27686/files?diff=split&w=0#diff-7a4d7b8fd8b4b5dba66053b430d042672ade65661c17043e1d4d100d59b70c7bR35) is the one in charge of producing the new screenshots in the this PR.